pylint/checkers/similar.py

```diff
@@ -919,6 +919,8 @@ def Run(argv=None):
             ignore_signatures = True
     if not args:
         usage(1)
+    if min_lines == 0:
+        sys.exit(0)
```

Comment:
Aren't we supposed to still run all the other checkers ? 
```suggestion
    if min_lines == 0:
        return
```

Reply:
I based this on L931 which also exits the checker with `sys.exit()`.

Reply:
`sys.exit()` is correct, as `def Run()` is run if the module is executed directly.